src/Command/AddApprenticeship/AddApprenticeshipCommand.cs

```diff
@@ -12,15 +12,14 @@ public class AddApprovalCommand : ICommand
         public string LegalEntityName { get; set; }
         public DateTime? ActualStartDate { get; set; }
         public DateTime PlannedEndDate { get; set; }
-        public decimal AgreedPrice { get; set; }
+        public decimal TotalPrice { get; set; }
```

Comment:
Any reason why this isn't nullable but the other price fields are? I would have thought all should be NON-nullable